spec/API_specification/array_api/__init__.py

```diff
@@ -12,3 +12,9 @@
 from .statistical_functions import *
 from .utility_functions import *
 from . import linalg
+
+
+__array_api_version__: str = "2021.12"
```

Comment:
Q: Would this appear in all versions (2021.12 and draft)?

Reply:
Oh yeah good shout. I would like to keep this variable in the stubs to keep in line with how we document all over non-function objects - we can bump it when we have a release, and just hide the value in the RST docs.

Reply:
I've assigned this var to `"YYYY.MM"`, which I think works well. And it means we don't have to update this variable, which we'll inevitably forget to do (without tooling at least).

---

Ideally I thought having no value would help, i.e.

```python
__array_api_version__: str
```

but this means we don't actually have a Python object `__array_api_version__` to import.

We could also play around with Sphinx `:autosummary:` to just hide the render value, but couldn't find an obvious and simple solution.

Reply:
Thanks, @honno, that's fine, I can live with it 🙂 I must have done some kind of Sphinx hacks to do what you described, but this is the kind of hacks that you forget once it's working... We can revisit if needed.